src/components/periodic-report/dto/periodic-report.dto.ts

```diff
@@ -53,6 +53,12 @@ class PeriodicReport extends Resource {
   @Field()
   readonly receivedDate: SecuredDateNullable;
 
+  @Field()
+  readonly skipped: boolean;
+
+  @Field()
+  readonly skippedReason?: string;
```

Comment:
These should be secured, and unfortunately that means right now that all the role files need to be updated to allow these two new props.